Assets/HoloToolkit/Input/README.md

```diff
@@ -1,5 +1,9 @@
 ## [Input]()
-Scripts that leverage the HoloLens input features namely Gaze, Gesture and Voice.
+Input System Diagrams:
+![alt text](https://cloud.githubusercontent.com/assets/8505814/19321359/98bbeca0-9069-11e6-9e17-e9845c8846f0.png)
```

Comment:
Please host images [here](https://github.com/Microsoft/HoloToolkit-Unity/tree/master/External/ReadMeImages).
You can create a sub-folder called Input. This system will be more manageable and transparent than an external website.